README.md

```diff
@@ -183,6 +183,45 @@ This library also supports Shopify's new [GraphQL API](https://help.shopify.com/
 result = shopify.GraphQL().execute('{ shop { name id } }')
 ```
 
+You can perform more complex operations using the `variables` and `operation_name` parameters of `execute`.
+
+For example, this GraphQL document uses a fragment to construct two named queries - one for a single order, and one for multiple orders:
+
+```graphql
+    # ./order_queries.graphql
+
+    fragment orderInfo on Order {
```

Comment:
```suggestion
    fragment OrderInfo on Order {
```

Reply:
We should be good to go now! For some reason it didn't show up until I did a pull + push.